pkg/apis/v1alpha3/types.go

```diff
@@ -46,11 +46,15 @@ type LogType string
 const (
 	// FileLogType defines the log type for logs stored in the file system.
 	FileLogType LogType = "File"
+
 	// S3LogType defines the log type for logs stored in the S3 object storage or S3 compatible alternatives.
 	S3LogType LogType = "S3"
 
 	// GCSLogType defines the log type for logs stored in the GCS object storage or GCS compatible alternatives.
 	GCSLogType LogType = "GCS"
+
+	// LokiLogType defines the log type for logs stored in the Loki.
+	LokiLogType LogType = "Loki"
```

Comment:
Just a note. I think it makes sense to make all these values case insensitive. It's more user friendly and there is no chance of name collisions.

Reply:
Yes. I was thinking of using strings.lower so that a user can send anything.